test/framework/functional/write.go

```diff
@@ -121,3 +121,17 @@ func (f *CollectorFunctionalFramework) WriteMessagesToLog(msg string, numOfLogs
 	log.V(3).Info("WriteMessagesToLog", "namespace", f.Pod.Namespace, "result", result, "err", err)
 	return err
 }
+
+// WriteMessagesWithNotUTF8SymbolsToLog write 12 symbols in ISO-8859-1 encoding
+func (f *CollectorFunctionalFramework) WriteMessagesWithNotUTF8SymbolsToLog() error {
+	filename := fmt.Sprintf("%s/%s_%s_%s/%s/0.log", fluentdLogPath[applicationLog], f.Pod.Namespace, f.Pod.Name,
+		f.Pod.UID, constants.CollectorName)
+	logPath := filepath.Dir(filename)
+	cmd := fmt.Sprintf("mkdir -p %s; echo \"$(ruby -e 'bad_chars = Array.new; bad_chars = [0xC0, 0xC1, 0xF5, "+
```

Comment:
There should be no need to initialize the array in this way: 
``` 
$ irb
2.7.2 :002 > bad_chars = [0xC0, 0xC1]
 => [192, 193] 

```

Reply:
Thanks, will fix. Don't understand why `loki/audit_logs_test.go` failed, something strange